db/migrate/20190721113740_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
```

Comment:
I would add some defaults here - just so that you see the syntax and see how it works